lib/screens/downloads_screen.dart

```diff
@@ -36,8 +38,16 @@ class DownloadsScreen extends StatelessWidget {
               const Divider(),
             ]),
           ),
-          const DownloadedItemsList(),
-          // CurrentDownloadsList(),
+          DownloadedItemsTitle(title: localizations.artists),
+          const DownloadedItemsList(type: BaseItemDtoType.artist),
+          DownloadedItemsTitle(title: localizations.albums),
+          const DownloadedItemsList(type: BaseItemDtoType.album),
+          DownloadedItemsTitle(title: localizations.playlists),
+          const DownloadedItemsList(type: BaseItemDtoType.playlist),
+          DownloadedItemsTitle(title: localizations.genres),
+          const DownloadedItemsList(type: BaseItemDtoType.genre),
+          DownloadedItemsTitle(title: localizations.tracks),
+          const DownloadedItemsList(type: BaseItemDtoType.track),
```

Comment:
Weird idea, but how about we order these categories i the same order as the tabs on the music screen?

Reply:
The reason I put artists first is that they have that extra sync button and are usually much less, so easier to get past. I'd also argue that the use-case is different. On the music screen, you want to get quickly to something to listen to, so typically albums and contained songs. On the downloads screen, my sorting idea was to start with the least amount of elements/"outermost" entity. Technically, playlists could be moved to the start.

Reply:
You are making a lot of assumptions about users there ^^

I actually mostly download artists and playlists, and very rarely albums. And my first tab are tracks ^^

The current order is fine, but another order probably isn't worse either. Right now it simply is arbitrary, and sorting based on tabs would at least be less arbitrary.

But it's really not a huge deal :)

Reply:
Oh, so you meant sorting based on the user preferences, not the default value of the tabs. That sounds reasonable enough. And true, those are definitely assumptions, that's why the music screen can be reordered, thankfully 😁 
I'd say we should keep it as is, and I'll look into filtering with chips after this PR is merged.

Reply:
Yes, I meant applying the user-configured order :)  
Alright, that sounds good. You might also wanna take a look at the #design channel on Discord for inspiration. Or what Spotify is doing with its Songs/Podcasts filter